TrustModel.md

```diff
@@ -0,0 +1,106 @@
+# Trust Model for Confidential Containers
+A clear definition of trust for the confidential containers project is needed to ensure the 
+components and architecture deliver the security principles expected for cloud native 
+confidential computing. It provides the solid foundations and unifying security principles 
+against which we can assess architecture and implementation ideas and discussions.
+
+The [Trust Modeling for Security Architecture Development article](https://www.informit.com/articles/article.aspx?p=31546) 
+defines Trust Modeling as :
+
+    A trust model identifies the specific mechanisms that are necessary to respond to a specific 
+    threat profile.
+
+    A trust model must include implicit or explicit validation of an entity's identity or the 
+    characteristics necessary for a particular event or transaction to occur.
+
+ The trust model also helps determine the location and direction of the trust boundaries where a 
+[trust boundary](https://en.wikipedia.org/wiki/Trust_boundary) describes a location where 
+ program data or execution changes its level of "trust", or where two principals with different 
+ capabilities exchange data or commands. Specific to Confidential Containers is the trust 
+ boundary that corresponds to the boundary of the Trusted Execution Environment (TEE). Generally, 
+ the TEE side of the trust boundary will be hardened to prevent the violation of the trust 
+ boundary.
+
+In order to describe and understand particular threats we need to establish trust boundaries and 
+trust models relating to the key aspects, components and actors involved in Cloud Native 
+Confidential Computing. We explore trust using different orthogonal ways of considering cloud 
+native approaches when they use an underlying TEE technology and 
+identifying where there may be considerations to preserve the value of using a TEE.
+
+<!--
+   Follow on pull requests will list here subsequent sections corresponding to orthogonal ways 
+of considering cloud native approaches
+Specifically including
+   - Personas
+   - Lifecycle
+   - TEE Boundary Location
+   - Secured v unsecured TEE Base Image
+-->
+
+Further documentation will highlight specific threat vectors in detail, considering risk, 
```

Comment:
Can we list the basic threat vectors at this stage, or reference another file (even a "stub" markdown file that could be created with this PR)?

Reply:
I added a stub file which has a link to the Issue #29 which holds a list to be documented